__tests__/store.test.js

```diff
@@ -0,0 +1,68 @@
+import AR from 'active-redux';
+import Store from 'active-redux/store';
+import mockStore from 'fixtures/store';
+
+describe('Store', () => {
+  beforeEach(() => {
+    AR.bind(mockStore);
+  });
+
+  const Comment = AR.define('comments', class Comment {});
+  const Person = AR.define('people', class Person {});
+
+  describe('#store', () => {
+    test('is the bound store', () => {
+      expect(Store.store).toBe(mockStore);
+    });
+  });
+
+  describe('#dispatch', () => {
+    test('provides a getter for store dispatch', () => {
+      expect(Store.dispatch).toBe(mockStore.dispatch);
+    });
+  });
+
+  describe('#state', () => {
+    test('provides a getter for store dispatch', () => {
+      expect(Store.state).toBe(mockStore.getState().api);
+    });
+  });
+
+  describe('#all', () => {
+    test('should return all values from a resource store', async () => {
+      const response = await Store.all({ model: Comment });
```

Comment:
This will keep Registry logic out of the store completely and make it a dumber class. I think there may be an additional layer of abstraction missing somewhere here.